xrpl/account/transaction_history.py

```diff
@@ -2,16 +2,17 @@
 from typing import Any, Dict, List, cast
 
 from xrpl.clients import Client, XRPLRequestFailureException
+from xrpl.core.addresscodec import is_valid_xaddress, xaddress_to_classic_address
 from xrpl.models.requests import AccountTx
 from xrpl.models.response import Response
 
 
-def get_latest_transaction(account: str, client: Client) -> Response:
+def get_latest_transaction(address: str, client: Client) -> Response:
     """
     Fetches the most recent transaction on the ledger associated with an account.
 
     Args:
-        account: the classic address of the account.
+        address: the account to query.
```

Comment:
Does this count as a breaking change because people might be using keyword args?

Reply:
yes and i don't see the need to change the variable name here.